### ISSUE TITLE
How to disable back gesture on main views

### ISSUE THREAD

**Issue by jan-kiszka** [OpenLauncherTeam/openlauncher#703]
I started to enjoy OpenLauncher on my new phone, but one detail is a bit annoying: The back gesture is interfering with sliding desktops left an right.

First experiments with the [gesture exclusion API](https://developer.android.com/codelabs/gesture-navigation#5) were promising - except that I quickly ran into that 200 dpi limitation when applying that on a per-view basis. I tried a lot but couldn't find out how to establish that screen-wide exclusions the API is also [talking about ("The limit does not apply [...] to the input method and home activity")](https://developer.android.com/reference/android/view/View#setSystemGestureExclusionRects(java.util.List%3Candroid.graphics.Rect%3E)). The Pixel Launcher can apparently do that, also without stickily mode, but I can't inspect how it achieves that. Any suggestions?

**Comment by gsantner** [comment]
OpenLauncher is not active developed. If you need something, you need to
do it on your own or find somebody to do it for you. Merge requests welcome.

**Comment by jan-kiszka** [comment]
@gsantner Do you have a bot running generating these replies?

I'm happy to write that MR, already built my own version. But I'm not Android guru to know all the tricks, that's why I'm asking here.

**Comment by gsantner** [comment]
No, I manually reply. But not sure how long I'm keep doing that. Developer contribution  interest is very low and user demand is high. 

I came here as a major contributor and  I got left alone in this project soon after.

**Comment by jan-kiszka** [comment]
Did the community moved to other OSS launcher - or gave up on that idea completely?

I'm used to fixing things and digging into stuff that is interesting. But I'm also overloaded with own under-maintained OSS projects. That's why I prefer the power of idea exchange over trying out stuff on my own for hours or even longer.

**Comment by gsantner** [comment]
Can't speak for others, but ask is high, do is low here^^
